transaction.go

```diff
@@ -6,29 +6,27 @@ import (
 	"github.com/lionelbarrow/braintree-go/customfields"
 )
 
-type TransactionStatus string
-
 const (
-	TransactionStatusAuthorizationExpired   TransactionStatus = "authorization_expired"
-	TransactionStatusAuthorizing            TransactionStatus = "authorizing"
-	TransactionStatusAuthorized             TransactionStatus = "authorized"
-	TransactionStatusGatewayRejected        TransactionStatus = "gateway_rejected"
-	TransactionStatusFailed                 TransactionStatus = "failed"
-	TransactionStatusProcessorDeclined      TransactionStatus = "processor_declined"
-	TransactionStatusSettled                TransactionStatus = "settled"
-	TransactionStatusSettlementConfirmed    TransactionStatus = "settlement_confirmed"
-	TransactionStatusSettlementDeclined     TransactionStatus = "settlement_declined"
-	TransactionStatusSettlementPending      TransactionStatus = "settlement_pending"
-	TransactionStatusSettling               TransactionStatus = "settling"
-	TransactionStatusSubmittedForSettlement TransactionStatus = "submitted_for_settlement"
-	TransactionStatusVoided                 TransactionStatus = "voided"
-	TransactionStatusUnrecognized           TransactionStatus = "unrecognized"
+	TransactionStatusAuthorizationExpired   string = "authorization_expired"
+	TransactionStatusAuthorizing            string = "authorizing"
+	TransactionStatusAuthorized             string = "authorized"
+	TransactionStatusGatewayRejected        string = "gateway_rejected"
+	TransactionStatusFailed                 string = "failed"
+	TransactionStatusProcessorDeclined      string = "processor_declined"
+	TransactionStatusSettled                string = "settled"
+	TransactionStatusSettlementConfirmed    string = "settlement_confirmed"
+	TransactionStatusSettlementDeclined     string = "settlement_declined"
+	TransactionStatusSettlementPending      string = "settlement_pending"
+	TransactionStatusSettling               string = "settling"
+	TransactionStatusSubmittedForSettlement string = "submitted_for_settlement"
+	TransactionStatusVoided                 string = "voided"
+	TransactionStatusUnrecognized           string = "unrecognized"
```

Comment:
If we're removing the type from these constants, we may as well make them untyped instead of `string`. Simply remove the `string` part of each line and it will be an untyped string.